src/govuk/components/notification-banner/notification-banner.js

```diff
@@ -37,8 +37,14 @@ NotificationBanner.prototype.setFocus = function () {
   }
 
   // Set tabindex to -1 to make the element focusable with JavaScript.
+  // Remove the tabindex on blur as the component doesn't need to be focusable after the page has
+  // loaded.
   if (!$module.getAttribute('tabindex')) {
     $module.setAttribute('tabindex', '-1')
+
+    $module.addEventListener('blur', function () {
+      $module.removeAttribute('tabindex')
```

Comment:
I'm curious if removing the `tabindex` attribute immediately after focusing it does the same job, although I don't think there's anything wrong with this approach – it just feels like it would be _ever so slightly_ simpler, and avoids having an event listener hanging around.

Not sure it's worth spending any more time on though – what do you think?

Reply:
Good question! Do you mean that we would just have:
```
 $module.focus()
 $module.removeAttribute('tabindex')
```

In my testing the focus isn't set if you do that. I'd be interested in understanding why the focus hasn't finished setting by the time you remove the tabindex 🤓  but like you said it might not be worth spending more time on.